src/holodeck/sensors.py

```diff
@@ -593,7 +593,7 @@ class AbuseSensor(HolodeckSensor):
     **Configuration**
 
     - ``AccelerationLimit``: Maximum acceleration the agent can endure from before
```

Comment:
"Maximum acceleration the agent can endure from" sounds strange&mdash;maybe I'm not reading it right. Maybe just "Maximum acceleration the agent can endure"?

Reply:
Deleted word